src/Drivers/AbstractSMS.php

```diff
@@ -6,6 +6,9 @@
 
 abstract class AbstractSMS
 {
+
+    protected $debug;
```

Comment:
It is dry run Feature which does not actually send any sms... but we will move it to the driver. Just thought it might be supported by other drivers as well